lib/cc/cli/init.rb

```diff
@@ -64,9 +64,8 @@ def create_default_engine_configs
       end
 
       def available_engine_configs
-        engine_names = existing_cc_config.engines.select do |_, config|
-          config.enabled?
-        end.keys
+        engines = existing_cc_config.engines || {}
```

Comment:
`existing_cc_config` returns nil when the file doesn't exist, right? So calling `engines` on that would raise an error?